test/Makefile

```diff
@@ -15,6 +15,10 @@ test: run
 # directory (keeping with the `wasi-libc` conventions).
 OBJDIR ?= $(CURDIR)/build
 DOWNDIR ?= $(CURDIR)/download
+# Like the top-level `wasi-libc` Makefile, here we can decide whether to test
+# with threads (`posix`) or without (`single`). IMPORTANT: the top-level include
```

Comment:
Feels like yet another argument in favor of [cmake](https://github.com/WebAssembly/wasi-libc/pull/330)